doc_source/using-s3-access-logs-to-identify-requests.md

```diff
@@ -285,4 +282,4 @@ If you are using Amazon S3 AWS CloudTrail logs, see [Using AWS CloudTrail to Ide
 
 ## Related Resources<a name="s3-access-logs-requests-more-info"></a>
 + [Amazon S3 Server Access Log Format](LogFormat.md)
-+ [Querying AWS Service Logs](https://docs.aws.amazon.com/athena/latest/ug/querying-AWS-service-logs.html)
\ No newline at end of file
++ [Querying AWS Service Logs](https://docs.aws.amazon.com/athena/latest/ug/querying-AWS-service-logs.html)
```

Comment:
Not sure what's the change here, maybe the line ending.
I edited the file directly on GitHub, didn't manually change this line.